src/data/dsp/data.json

```diff
@@ -1 +1 @@
wer-station":1},"time":10,"producers":["assembler-1","assembler-2","assembler-3"]},{"id":"storage-tank","name":"Storagetank","in":{"iron-plate":8,"stone-brick":4,"glass":4},"out":{"storage-tank":1},"time":2,"producers":["assembler-1","assembler-2","assembler-3"]},{"id":"deuterium-formula","name":"Deuteriumfractionation","in":{"hydrogen":0.01},"out":{"deuterium":0.01},"time":1,"producers":["fractionator"]},{"id":"iron-vein","mining":true,"time":2,"out":{"iron-ore":1},"producers":["mining-drill"]},{"id":"copper-vein","mining":true,"time":2,"out":{"copper-ore":1},"producers":["mining-drill"]},{"id":"silicium-vein","name":"Siliconvein","mining":true,"time":2,"out":{"silicium-ore":1},"producers":["mining-drill"]},{"id":"titanium-vein","mining":true,"time":2,"out":{"titanium-ore":1},"producers":["mining-drill"]},{"id":"stone-vein","mining":true,"time":2,"out":{"stone-ore":1},"producers":["mining-drill"]},{"id":"coal-vein","mining":true,"time":2,"out":{"coal-ore":1},"producers":["mining-drill"]},{"id":"fireice-vein","mining":true,"time":2,"out":{"gas-hydrate":1},"producers":["mining-drill"]},{"id":"diamond-vein","mining":true,"time":2,"out":{"diamond-ore":1},"producers":["mining-drill"]},{"id":"fractal-vein","mining":true,"time":2,"out":{"fractal-silica":1},"producers":["mining-drill"]},{"id":"crysrub-vein","name":"Organiccrystalvein","mining":true,"time":2,"out":{"crystal-rubber":1},"producers":["mining-drill"]},{"id":"grat-vein","mining":true,"time":2,"out":{"grating-ore":1},"producers":["mining-drill"]},{"id":"bamboo-vein","name":"Spiniformstalagmite","mining":true,"time":2,"out":{"bamboo-crystal":1},"producers":["mining-drill"]},{"id":"mag-vein","mining":true,"time":2,"out":{"mono-mag-ore":1},"producers":["mining-drill"]},{"id":"oil-vein","mining":true,"time":1,"out":{"oil":1},"producers":["oil-extractor"]},{"id":"photon-capacitor-full","mining":true,"out":{"photon-capacitor-full":5},"time":60,"producers":["ray-receiver"]},{"id":"water-vein","mining":true,"time":1.2,"out":{"water":1},"producers":["water-pump"]},{"id":"sulphuric-acid-vein","mining":true,"time":1.2,"out":{"sulphuric-acid":1},"producers":["water-pump"]},{"id":"gas-giant","name":"Gasgiant","mining":true,"time":8,"out":{"hydrogen":4.08,"deuterium":0.17},"producers":["orbital-collector"]},{"id":"ice-giant","name":"Icegiant","mining":true,"time":8,"out":{"gas-hydrate":1.95,"hydrogen":0.84},"producers":["orbital-collector"]},{"id":"accumulator-full","name":"Accumulator(Full)","time":2,"in":{"accumulator":1},"producers":["energy-exchanger"]}],"defaults":{"modIds":[],"minBelt":"belt-1","maxBelt":"belt-3","fuel":"coal-ore","disabledRecipes":["ice-giant","nanotube-lv2","partical-capacitor-lv2","photo-shifter-lv2","silicium-high-purity-lv2","space-warper-lv2","casimir-crystal-lv2","diamond-lv2","gas-giant","graphene-lv2","crysrub-vein","fireice-vein","sulphuric-acid-vein"],"minFactoryRank":["assembler-1"],"maxFactoryRank":["assembler-3"],"moduleRank":[]},"limitations":{"productivity-module":[]}}
```

Comment:
It looks like your update removed all the spaces from strings. E.g. `position` fields went from `0px 0px` to `0px0px` and `name` from `Organic crystal` to `Organiccrystal`.